Jenkinsfile

```diff
@@ -1,12 +1,41 @@
 @Library('jenkins-shared-library') _
 
-/**
-* Job Parameters:
-*	skipTests - whether to skip unit tests during the build process or not
-*	skipDeploy - whether to deploy build artifacts in case of successful maven build or not (should be false by default)
-*	skipDownstreamBuilds - whether to skip execution of downstream builds
-*	downstreamBuild - name of downstream build
-**/
+properties(
+	[
+		disableConcurrentBuilds(),
+		parameters(
+			[
+				booleanParam(
+					name: 'skipTests',
+					defaultValue: true,
+					description: 'Skip running the entire test suite'
+				),
+				booleanParam(
+					name: 'skipDeploy',
+					description: 'Skip deploying artifacts at the end'
+				),
+				booleanParam(
+					name: 'skipDownstreamBuilds',
+					description: 'Skip execution of downstream builds'
+				),
+				hidden(
+					name: 'downstreamBuild',
+					defaultValue: 'snow-owl-ext'
+				),
+				hidden(
+					name: 'custom_maven_global_settings',
+					defaultValue: '895dc1f0-42a1-4b7d-8b6c-20f93e45e9b8'
+				),
+				hidden(
+					name: 'custom_maven_settings',
+					defaultValue: '020e9686-ed90-4a75-9549-57ea85b5cd54'
+				)
```

Comment:
Can these be set to empty defaults in the OSS repository? They aren't likely to be used with the same values in build environments other than ours.